src/components/ResultCard.js

```diff
@@ -0,0 +1,158 @@
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import {
+  CardWrapper,
+  CardImage,
+  CardContent,
+  CardHeading,
+  CardText,
+  CardButtonGroup,
+  CardButton,
+} from './Card';
+import { Link, Text } from '@chakra-ui/core';
+
+// TODO: Replace with real fallback images for each category.
+// This should all probably be defined in the database somewhere, eh?
+const categoryData = {
+  entertainment: {
+    label: 'Entertainment',
+    image: {
+      src: 'https://source.unsplash.com/random',
+      alt: 'Id facilisis dictum consequat sit orci.',
+    },
+    buttonText: 'Learn more',
+  },
+  foodAndBeverage: {
+    label: 'Food and Beverage',
+    image: {
+      src: 'https://source.unsplash.com/random',
+      alt: 'Id facilisis dictum consequat sit orci.',
+    },
+    buttonText: 'Order',
+  },
+  healthAndWellness: {
+    label: 'Health and Wellness',
+    image: {
+      src: 'https://source.unsplash.com/random',
+      alt: 'Id facilisis dictum consequat sit orci.',
+    },
+    buttonText: 'Learn more',
+  },
+  professionalServices: {
+    label: 'Professional Services',
+    image: {
+      src: 'https://source.unsplash.com/random',
+      alt: 'Id facilisis dictum consequat sit orci.',
+    },
+    buttonText: 'Contact',
+  },
+  retail: {
+    label: 'Retail',
+    image: {
+      src: 'https://source.unsplash.com/random',
+      alt: 'Id facilisis dictum consequat sit orci.',
+    },
+    buttonText: 'Shop',
+  },
+};
+
+const ResultCard = forwardRef(
+  (
+    {
+      children,
+      imageSrc,
+      imageAlt,
+      businessCategory,
+      businessName,
+      businessDescription,
+      businessLocation,
```

Comment:
It seems a tad redundant to have `business` in front of each word. Some options to make the code a little less verbose could be an **object with all of the business stuff** in it (i.e. `business.category`, `business.name`) and that could _also apply to things like the image_ (i.e. `image.src`, `image.alt`).

Or another option could just be having the unique names be without the first part (i.e. `src`, `alt`, `category`, `name`) but that could be confusing.

Reply:
Prefer to stay away from object literal props as a general rule if I can, but happy to drop the business prefix.